### PR TITLE
feat: Partners section

### LINKED ISSUE #75: CODING:develop the partners section in the project page.
### Description
As a developer,
I need the following section of mockup to be developed,
so that I can add it in my projects page.

https://xd.adobe.com/view/e83904d0-fe4d-47c1-7762-155628f7de25-9e08/

### Mocks
image

**SIZE AND OTHER markings**
image

**here is a gif**
Webp net-gifmaker (1)

Font Size for the the heading:  35pt for partners 29pt for various types weight- medium
Font Style- Avenir
Font Color (heading): #010101
Font Color (Body): #103B81

### Acceptance Criteria
**Update [Required]**

- [x] create a hover style section of projects
- [x] Follows all the design guidelines as mentioned and is responsive enough on large screen devices.
- [x] create all the partners icons(total around 90 are there) all are mentioned in the link in both format grayscale as well as in color 

### Definition of Done

- [x]  All of the required items are completed.
- [x]  Approval by 2 mentor.

### Estimation
6-7 hours

### PR DESCRIPTION
**Pull request by annabauza**
### Description

Adding Partners section to projects screen. This section is currently commented out and will be published as soon as we have list of partners.

Fixes #75

### Type of Change:
**Delete irrelevant options.**

- Code

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

ezgif com-video-to-gif-3

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (designs)
- [x] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by nandini45** [comment]
@annabauza  i think can we align all the icons somehow means both sides .
the  gap should be equal and we cab place several icons
something like this
image

**Comment by annabauza** [comment]
@nandini45 which gap? Gutters are equal

**Comment by annabauza** [comment]
@anitab-org/coding-team can anyone please review this?

**Comment by annabauza** [comment]
@isabelcosta ☝️

**Comment by annabauza** [comment]
This sponsor where taken from anitab portal, are they fake?

**Comment by nandini45** [comment]
for sure they aren't fake .. all the partners are specifically mentioned on Anitab.org website

and i think just putting any company name without being associated is not legal.

**Comment by isabelcosta** [comment]
@annabauza @nandini45 if portal also has it, I guess its fine. Its just that I haven't seen any sponsor/partner talk regarding this open source community. @mayburgos may clarify this better for us. I haven't thought about AnitaB.org partners 🤔 and if they apply to us.

**Comment by annabauza** [comment]
@isabelcosta currently this section is comment out and does not show any Partners. If you can please review other PRs are waiting for this PR. Thanks.

File: src/Components/Projects/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  return (
+    <View
+      style={{
+        width: '80%',
+        alignItems: 'left',
+        paddingLeft: 16,
+        paddingRight: 16,
+      }}
+    >
+      {/* <SectionHeader title="Partners" />
```
Comment (annabauza) [inline src/Components/Projects/index.js:16]: @isabelcosta this section has been commented out.

**Comment by isabelcosta** [comment]
> @isabelcosta currently this section is comment out and does not show any Partners. If you can please review other PRs are waiting for this PR. Thanks.

Can you tag me on those PRs or is it all of them?

**Comment by annabauza** [comment]
@isabelcosta sorry I express myself wrong. Can you review this PR because other PRs waiting for this to be merged, people are duplicating work for headers and subheaders. Thanks

File: src/Components/Projects/index.js (hunk 1 above)
Comment (isabelcosta) [reply src/Components/Projects/index.js:16]: If this Sponsors section is commented, could you remove the 50+ binary files from the PR, or do you think it's beneficial to keep them in the git history? @annabauza

**Pull request merged**